remote-config/src/main/kotlin/com/rakuten/tech/mobile/remoteconfig/api/ConfigApiClient.kt

```diff
@@ -47,20 +53,53 @@ internal class ConfigApiClient @VisibleForTesting constructor(
         throw InvalidRemoteConfigBaseUrlException(exception)
     }
 
-    fun fetchPath(path: String): Response {
-        val url = requestUrl.newBuilder()
-            .addPathSegments(path)
-            .build()
+    fun fetchPath(path: String, isConfig: Boolean): Response {
+        val builder = requestUrl.newBuilder().addPathSegments(path)
 
-        return client.newCall(
-            Request.Builder()
-                .url(url)
+        if (isConfig) {
+            try {
+                val pInfo: PackageInfo = context.packageManager.getPackageInfo(context.packageName, 0)
+                builder.addQueryParameter(PARAM_KEY_APP_VERSION, pInfo.versionName)
+            } catch (e: PackageManager.NameNotFoundException) {
+                Log.d(SDK_NAME, e.localizedMessage.toString())
+            }
+
+            builder.addQueryParameter(PARAM_KEY_OS_VERSION, Build.VERSION.SDK_INT.toString())
```

Comment:
Did you check on what the backend is expecting for this value? I believe it's supposed to be the version name, rather than the SDK version. i.e. `Build.VERSION.RELEASE` (8.0, 9.0, etc.)

Reply:
Changed to `Build.VERSION.RELEASE`